code/modules/mining/equipment/kinetic_crusher/trophies/lavaland_mob_trophies.dm

```diff
@@ -0,0 +1,208 @@
+	AddComponent(/datum/component/crusher_damage_ticker, APPLY_WITH_SPELL, bonus_value)
+
+/obj/item/crusher_trophy/goliath_tentacle/effect_desc()
+	return "mark detonation to do <b>[bonus_value]</b> more damage for every <b>[missing_health_desc]</b> health you are missing"
+
+/obj/item/crusher_trophy/goliath_tentacle/on_mark_detonation(mob/living/target, mob/living/user)
+	var/missing_health = user.maxHealth - user.health
+	missing_health *= missing_health_ratio //bonus is active at all times, even if you're above 90 health
+	missing_health *= bonus_value //multiply the remaining amount by bonus_value
+	if(missing_health > 0)
+		target.adjustBruteLoss(missing_health) //and do that much damage
+		log_combat(user, target, "struck for bonus damage with a trophy ", src, "crusher damage")
+		SEND_SIGNAL(src, COMSIG_CRUSHER_SPELL_HIT, target, user, missing_health)
+
+/**
+ * Watcher
+ * Detonating a mark causes the victim's next ranged attack/special attack to cool down slower.
+ */
+/obj/item/crusher_trophy/watcher_wing
+	name = "watcher wing"
+	desc = "A wing ripped from a watcher. Suitable as a trophy for a kinetic crusher."
+	icon_state = "watcher_wing"
+	denied_type = /obj/item/crusher_trophy/watcher_wing
+	bonus_value = 5
+
+/obj/item/crusher_trophy/watcher_wing/effect_desc()
+	return "mark detonation to prevent certain creatures from using certain attacks for <b>[bonus_value * 0.1]</b> second\s"
+
+/obj/item/crusher_trophy/watcher_wing/on_mark_detonation(mob/living/target, mob/living/user)
+	if(ishostile(target))
+		var/mob/living/simple_animal/hostile/victim = target //this is fun, doesn't apply to basic mobs. need to look into that
+		if(victim.ranged) //briefly delay ranged attacks
+			if(victim.ranged_cooldown >= world.time)
+				victim.ranged_cooldown += bonus_value
+			else
+				victim.ranged_cooldown = bonus_value + world.time
+
+/**
+ * Watcher - icewing variant
+ * Detonating a mark causes the victim's next ranged attack/special attack to cool down even slower.
+ */
+/obj/item/crusher_trophy/watcher_wing/ice_wing
+	name = "icewing watcher wing"
+	desc = "A carefully preserved frozen wing from an icewing watcher. Suitable as a trophy for a kinetic crusher."
+	icon_state = "ice_wing"
+	bonus_value = 8
+
+/**
+ * Watcher - magmawing variant
+ * Detonating a mark causes the next destabilizer shot to deal damage.
+ * The bonus lasts for a single shot and resets after 30 seconds.
+ */
+/obj/item/crusher_trophy/blaster_tubes/magma_wing
+	name = "magmawing watcher wing"
+	desc = "A still-searing wing from a magmawing watcher. Suitable as a trophy for a kinetic crusher."
+	icon_state = "magma_wing"
+	gender = NEUTER
+	bonus_value = 5
+
+/obj/item/crusher_trophy/blaster_tubes/magma_wing/effect_desc()
+	return "mark detonation to make the next destabilizer shot deal <b>[bonus_value]</b> damage"
+
+/obj/item/crusher_trophy/blaster_tubes/magma_wing/on_projectile_fire(obj/projectile/destabilizer/marker, mob/living/user)
+	if(deadly_shot)
```

Comment:
```suggestion
	if(!deadly_shot)
		return
```
I almost wonder if for this & the other one you could just replace the marker shot with a different projectile somehow... this is probably fine though.